agimus-demos/Makefile

```diff
@@ -0,0 +1,22 @@
+# robotpkg Makefile for:	wip/agimus-vision
+# Created:			Long Ha Thuc on Tue, 18 Aug 2020
+#
+
+ORG=			agimus
+NAME=			agimus-demos
+VERSION=		1.1.0
+DOWNLOAD_LOC=   releases/download
+MASTER_SITE_AGIMUS=     https://github.com/agimus/
+DISTNAME=		${NAME}-${VERSION}
+
+MASTER_SITES=		${MASTER_SITE_AGIMUS:=${NAME}/${DOWNLOAD_LOC}/v${VERSION}/}
+MASTER_REPOSITORY=	${MASTER_REPOSITORY_GITHUB}${ORG}/${NAME}/${DOWNLOAD_LOC}/v${VERSION}
```

Comment:
`MASTER_REPOSITORY` should point to a .git tree: it is used for `make checkout`. Here, you can use:
```
MASTER_REPOSITORY?=	${MASTER_REPOSITORY_GITHUB}${ORG}/${NAME}.git
```